chapters/fr/_toctree.yml

```diff
@@ -191,7 +191,11 @@
     title: Quiz de fin de chapitre
     quiz: 9
 
-- title: Evènements liés au cours d'Hugging Face
+- title: Evènements liés au cours
   sections:
   - local: event/1

```

Comment:
turbo nit: I renamed this folder to `events` in #329 - would you mind doing the same please?

Reply:
@lewtun  Done